src/helpers/convertUnits.ts

```diff
@@ -0,0 +1,21 @@
+type Value = {
+	amount: string;
+	unit: string;
+};
+
+export const convertUnit = ({ amount: amountArg, unit }: Value): Value => {
+	let amount = Number(amountArg);
+	if (unit === 'grams') {
+		if (amount > 1000) {
+			amount = amount / 1000;
+			unit = 'kilograms';
+		}
+	}
+	if (unit === 'milliliters') {
```

Comment:
units can be moved to enums

Reply:
I have unit group by weight, volume and I can't not create nested enum